modules/core/src/main/java/org/locationtech/jts/io/WKTWriter.java

```diff
@@ -150,15 +159,7 @@ public static String toLineString(Coordinate p0, Coordinate p1)
    *      s without scientific notation.
    */
   private static DecimalFormat createFormatter(PrecisionModel precisionModel) {
```

Comment:
return `OrdinateFormat`?

Reply:
That would be better... It would just require a lot more changes throughout WKTWriter.  I will consider...

Reply:
I have refactored `WKTWriter` to use `OrdinateFormat`.  This removes the dependency on `DecimalFormat`.